### PR TITLE
realsense Depth comparison

### PR DESCRIPTION
**Pull request by Erol444**
Target (color img):
image

Without active stereo
image

With active stereo
image

### CODE REVIEW

**Comment by Erol444** [comment]
Long range depth:
image

File: docs/source/pages/guides/realsense_comparison.rst [hunk 1]
```diff
@@ -12,6 +12,44 @@ In August of 2021, Intel announced it is winding down RealSense Camera Business
...
+  Laser dot projector enabled (active stereo)
+
+.. figure:: https://user-images.githubusercontent.com/18037362/184262380-9ad26b2f-0b31-439d-b887-5c89a2ad67bb.png
+  
+  Target (color image). Table and wall are featureless surfaces.
+
+Custom stereo depth perception
+##############################
+
+Our platform was built from ground up with the idea of being **customizable**. All our products based on :ref:`OAK SOM <System on Module Designs>`
```
Comment (stuartsmoore) [inline docs/source/pages/guides/realsense_comparison.rst:37]:
```suggestion
Our platform was built from the ground up with the idea of being **customizable**. All of our products based on :ref:`OAK SOM <System on Module Designs>`
```

File: docs/source/pages/guides/realsense_comparison.rst [hunk 2]
```diff
@@ -12,6 +12,44 @@ In August of 2021, Intel announced it is winding down RealSense Camera Business
...
+are opensource so you could easily redesign the board (see :ref:`Integrating DepthAI into products`), for example to **change
```
Comment (stuartsmoore) [inline docs/source/pages/guides/realsense_comparison.rst:38]:
```suggestion
are open-source so you can easily redesign the board (see :ref:`Integrating DepthAI into products`), for example to **change
```

File: docs/source/pages/guides/realsense_comparison.rst [hunk 3]
```diff
@@ -12,6 +12,44 @@ In August of 2021, Intel announced it is winding down RealSense Camera Business
...
+the stereo baseline distance** or use a different image sensor (we support :ref:`a bunch of different sensors <Supported sensors>`).
+
+With :ref:`OAK FFC line <Modular Cameras Designs>` is great for prototyping, as it allows users to use different camera sensors/optics
```
Comment (stuartsmoore) [inline docs/source/pages/guides/realsense_comparison.rst:41]:
```suggestion
:ref:`OAK FFC line <Modular Cameras Designs>` is great for prototyping, as it allows users to use different camera sensors/optics
```

File: docs/source/pages/guides/realsense_comparison.rst [hunk 4]
```diff
@@ -12,6 +12,44 @@ In August of 2021, Intel announced it is winding down RealSense Camera Business
...
+and place them at an ideal stereo baseline distance for their application.
+
+Below is a long-range disparity depth visualized over a color frame. Customer used narrow FOV M12 lenses
```
Comment (stuartsmoore) [inline docs/source/pages/guides/realsense_comparison.rst:44]:
```suggestion
Below is a long-range disparity depth visualized over a color frame. This customer used narrow FOV M12 lenses
```

**Pull request merged**